src/api/spec/README.md

```diff
@@ -144,18 +144,16 @@ Attention: Faker generates random but **NOT** unique data!
 
 ### Backend responses
 
-If you require a response from the OBS backend for your new test you need to
-start it with
-
-```
-docker-compose run --rm frontend bundle exec rake db:fixtures:load RAILS_ENV=test
-docker-compose run --rm frontend script/start_test_backend
-```
-
 We use [VCR](https://github.com/vcr/vcr) to record the response from the backend.
 VCR records the HTTP interactions with the backend and replays them during future test runs for fast, deterministic, accurate tests.
 Once your test ran successfully for the first time [VCR](https://github.com/vcr/vcr) will have recorded a new cassette (a simple yml file) in `spec/cassettes`.
 
+You can log into the frontend container which already has a backend running with: 
```

Comment:
And why would I?

Reply:
To run rspec inside the container ;) You mean I should add how I can run rspec inside the container?

Reply:
Well it's not clear what you are trying to say. And in general I would leave docker stuff out of this...

Reply:
what is your recommended way to update vcrs? The one @ChrisBr outlined, worked for me

Reply:
I'm not saying to use any other way. I'm saying that this document shouldn't include any docker specifics (only normal commands) and should refer to the wiki for how to run them in the development environment. 

That way we have one place to keep up to date.

Reply:
Ok! Just added it because before it was also already there! I will remove it and move it to our development setup wiki.